### ISSUE TITLE
Unhandled promise rejection errors if PHPMD returns no output

### ISSUE THREAD

**Issue by sandhje** [sandhje/vscode-phpmd#14]
See error message reported in issue #13.

**Comment by neerolyte** [comment]
This also happens if the extension tries to validate a file that hasn't been saved yet.

E.g

Creating a new empty file (`command-n`) and then setting the mode to PHP pops up:

![image](https://user-images.githubusercontent.com/1070702/28189801-ebb1be38-686a-11e7-80f4-20d320800508.png)

and logs:

```
(node:50252) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: An error occured, no output was received after executing the phpmd command
(node:50252) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by sandhje** [comment]
Thanks, I haven't tested that scenario yet. I'll fix that.

**Comment by sandhje** [comment]
Resolved in version 1.0.1
